### PR TITLE
Allow for different E2E test directories

### PR DESCRIPTION
**Pull request by mangelajo**
This will allow for a cleaner separation between the external network
tests and regular E2E tests, avoiding leaking of the skip/focus details
to other places.

Signed-off-by: Miguel Angel Ajo <majopela@redhat.com>

### CODE REVIEW

**Comment by mangelajo** [comment]
@mkimuram  ^

**Review by skitt — APPROVED** [review]

**Comment by mangelajo** [comment]
I'm breaking something, isn't the default working? https://github.com/submariner-io/shipyard/pull/521/checks?check_run_id=2304358600#step:3:989

It triggers the "test with subctl" somehow.

File: scripts/shared/e2e.sh [hunk 1]
```diff
@@ -6,9 +6,10 @@ source ${SCRIPTS_DIR}/lib/shflags
 DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
 DEFINE_string 'focus' '' "Ginkgo focus for the E2E tests"
 DEFINE_string 'skip' '' "Ginkgo skip for the E2E tests"
+DEFINE_string 'testdir' 'tests/e2e' "Directory under to be used for E2E testing"
```
Comment (mangelajo) [inline scripts/shared/e2e.sh]:
```suggestion
DEFINE_string 'testdir' 'test/e2e' "Directory under to be used for E2E testing"
```

Comment (mangelajo) [reply scripts/shared/e2e.sh]: found the issue

**Review by tpantelis — APPROVED** [review]

**Pull request merged**